src/store/ChartStore.js

```diff
@@ -282,6 +284,14 @@ class ChartStore {
         const studiesStore = this.mainStore.studies;
         stxx.callbacks.studyOverlayEdit = studiesStore.editStudy;
         stxx.callbacks.studyPanelEdit = studiesStore.editStudy;
+        const chartiqHome = stxx.home;
+        stxx.home = function () {
+            if (settings.historical) {
+                feed.onRangeChanged();
+            } else {
+                chartiqHome.apply(this, {});
```

Comment:
@brucebinary what I come to at last I tried to test `setRange`, but not work properly it didn't show correct range, you should scroll a little after setRange called, so I use feed range changed

Reply:
Consider using endEpoch value instead of historical setting. Instead of using setRange you can set scroll. Or both. 

You could just call chartiqHome().